homework-3/src/components/Forms/FormRestorePassword.js

```diff
@@ -25,31 +24,25 @@ const FormRestorePassword = () => {
         email: '',
       }}
       validate={(values) => {
-        const regulEmail =
-          /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
         if (!regulEmail.test(values.email)) {
-          setEmail((isEmail = true));
+          setEmail(!isEmail);
         } else {
-          setEmail((isEmail = false));
+          setEmail(!isEmail);
```

Comment:
You di the same logic in if, probably will work wrong, test it and correct.

Reply:
Added folder `constants` which has a file `index.js`